bin/init-aio.sh

```diff
@@ -58,7 +58,7 @@ if [ $IS_START_REST = "true" ]; then
     . /config/rest.env
     /usr/bin/neofs-rest-gw &
 
-    while [[ "$(curl -s -o /dev/null -w %{http_code} $REST_GW_SERVER_LISTEN_ADDRESS)" != "307" ]];
+    while [[ "$(curl -s -o /dev/null -w %{http_code} $REST_GW_SERVER_ENDPOINTS_0_ADDRESS)" != "307" ]];
```

Comment:
Doesnt rest-gw have some helathcheck?

Reply:
not in core protocol, only Prometheus and indirect signs like this